app/views/account/fragments/yourDetails.scala.html

```diff
@@ -23,12 +24,14 @@
         <dd class="mma-section__list--content">
             <div>@contact.firstName @contact.lastName</div>
             <div>@{
+                def opt(string: String) = Some(string).filter(_.trim.nonEmpty)
```

Comment:
How about tidying up the display and putting the trim inside the Some? e.g.

`Some(string.trim).filter(_.nonEmpty)`